analyses/molecular-subtyping-embryonal/01-samples-to-subset.Rmd

```diff
@@ -17,8 +17,14 @@ To summarize - we will identify biospecimen IDs that should be included for down
 1. An RNA-seq biospecimen includes a _TTYH1_ fusion (5' partner).
    We can consume the files from the [`fusion-summary` module](../fusion-summary/) for this step.
    (See also: [this comment on `AlexsLemonade/OpenPBTA-analysis#401`](https://github.com/AlexsLemonade/OpenPBTA-analysis/pull/401#issuecomment-573669727).)
-2. Any sample with `"Embryonal tumor"` in the `broad_histology` column of `pbta-histologies.tsv` that is not labeled `"Medulloblastoma"` or `"Atypical Teratoid Rhabdoid Tumor (ATRT)"` in `pathology_diagnosis` per [this comment on `AlexsLemonade/OpenPBTA-analysis#251`](https://github.com/AlexsLemonade/OpenPBTA-analysis/issues/251#issuecomment-568220913).
-   We'll look at the `integrated_diagnosis` column as well.
+   
+2. An RNA-seq biospecimen sample includes a _MN1_ fusion (5' partner) [per this comment](https://github.com/AlexsLemonade/OpenPBTA-analysis/pull/785#issuecomment-695015488).
+
+3. Any sample with "Supratentorial or Spinal Cord PNET" in the `pathology_diagnosis` column of the metadata `pbta-histologies.tsv` (also [per this comment](https://github.com/AlexsLemonade/OpenPBTA-analysis/issues/752#issuecomment-697000066)).
+
+4. Any sample with "Neuroblastoma" in the `pathology_diagnosis` column, `primary_site != "Other locations NOS"`, and `pathology_free_text_diagnosis` does not contain "peripheral".
```

Comment:
I would probably take out `primary_site != "Other locations NOS"` here. I would expect the code to perform filtering on the basis of `primary_site != "Other locations NOS"` the way this is written and that's not what the code does.